### ISSUE TITLE
(suggestion) Sublime "Select to Mark" functionality

### ISSUE THREAD

**Issue by abrowne2** [dreamfliper/vscode-mark-sublime#1]
Hi, cool extension.

A suggestion:

Sublime has an option to "select_to_mark", when you press a key on a specified line from your mark.

Thanks

**Comment by dreamfliper** [comment]
Yes, you're right. There's also a "Delete to mark" functionality.
I'll add these two features in the future.

Thanks for the feedback
